examples/noop_allocator.cpp

```diff
@@ -0,0 +1,18 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+#include <iostream>
+
+int main() {
+  auto &rm = umpire::ResourceManager::getInstance();
+  auto alloc = rm.getAllocator("NO_OP");
```

Comment:
NoOpResorceFactory.cpp currently is using "No_Op".  This string needs to be the same.  I suggest either using "NO_OP" or potentially "NOOP".